### DISCUSSION TITLE
Import from folder

### DISCUSSION THREAD

**Discussion by dm30** [johnfactotum/foliate#1341] [Ideas]
Foliate is beautiful, but adding books one at a time hampers its usefulness. Is any bulk or import feature on the pipeline?
Thank you.

**Comment by johnfactotum** [comment]
See #1196 and #730.

**Comment by dm30** [comment]
Thank you! I took a look. How is Tracker going on? 3 years have passed. A very basic 'add folder', drag and drop feature, or allowing the picker to select multiple items would do wonders to Foliate.
